docs/guide/pmndrs/tonemapping.md

```diff
@@ -0,0 +1,71 @@
+# ToneMapping
+
+<DocsDemo>
+  <ToneMappingDemo />
+</DocsDemo>
+
+The `ToneMapping` effect from the [`postprocessing`](https://pmndrs.github.io/postprocessing/public/docs/class/src/effects/ToneMappingEffect.js~ToneMappingEffect.html) package provides an abstraction for various tone mapping algorithms to improve the visual rendering of HDR (high dynamic range) content. Tone mapping is used to map high-range brightness values to a range that is displayable on standard screens. This effect contributes significantly to the visual quality of your scene by controlling luminance and color balance.
+
+::: info
+If the colors in your scene look incorrect after adding the EffectComposer, it might be because tone mapping is deactivated by default, which is normal behavior. Add `<ToneMapping>` manually as an effect at the end of the `<EffectComposer>` to fix this.
+:::
+
+## Usage
+
+The `<ToneMapping>` component is easy to set up and comes with multiple tone mapping modes to suit different visual requirements. Below is an example of how to use it in a Vue application.
+
+```vue{2,4,7-8,32-36}
```

Comment:
**suggestion:** While I like to have a full example on the page, I think it is better to stay uniform with the other effects and only show the usage.